### PR TITLE
Introduced a loop for to check current ali names and don't allow to a…

### PR DESCRIPTION
**Pull request by litosnano**
…dd a new one that have the same name

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -106,6 +106,15 @@ func details(c config.Config, name string) {
 }
 
 func add(c config.Config, path, name string) {
+
+
+	for _, c := range c.Commands {
```
Comment (adriacidre) [inline main.go]: Nice! As this is going to be used also for `remove` or `details` commands. I would say to move it inside `config.Config` struct something like:
```
func (c *Config) Get(name string) (Command , error) {
  ....
}
```

So, we can check if that command exists from the outside with:
```
command, err := c.Get(name)
if err != nil {
    fmt.Println("Sorry, this ali is already taken.")
    return
}
```

File: main.go [hunk 2]
```diff
@@ -106,6 +106,15 @@ func details(c config.Config, name string) {
...
+		if c.Name == name {
+			fmt.Println("Sorry, this ali is already taken.")
```
Comment (adriacidre) [inline main.go]: change the text for something like `alias already in use`

**Review by adriacidre — CHANGES_REQUESTED** [review]
Looks good! Let's see if we can improve and make the code reusable by refactoring :)

File: main.go (hunk 1 above)
Comment (litosnano) [reply main.go]: Ok. I'll move it to config.Config

File: main.go (hunk 2 above)
Comment (litosnano) [reply main.go]: OK. I'll do it

File: main.go [hunk 3]
```diff
@@ -123,6 +136,12 @@ func add(c config.Config, path, name string) {
 }
 
 func rm(c config.Config, path, name string) {
+	_, err := c.Get(name)
+	if err != nil {
+		fmt.Println("It's not posible remove "+name+" as it doesn't exist.")
```
Comment (adriacidre) [inline main.go:141]: I would change the text by `Alias not found` instead.

File: main.go [hunk 4]
```diff
@@ -82,6 +82,12 @@ func list(c config.Config) {
 }
 
 func details(c config.Config, name string) {
+	_, err := c.Get(name)
+	if err != nil {
+		fmt.Println("It's not posible to show "+name+" as it doesn't exist.")
```
Comment (adriacidre) [inline main.go]: I would change the text by `Alias not found` instead.

**Review by adriacidre — APPROVED** [review]
Looks good to me, I will merge it as soon as you change those `not found` outputs.

File: main.go (hunk 3 above)
Comment (litosnano) [reply main.go:141]: Great. I've just done it.

File: main.go (hunk 4 above)
Comment (litosnano) [reply main.go]: Great, I've just done it and push it.

**Review by adriacidre — APPROVED** [review]
looks good!

**Pull request merged**